ledger/participant-integration-api/src/test/lib/scala/platform/store/dao/JdbcLedgerDaoTransactionsSpec.scala

```diff
@@ -503,24 +503,24 @@ private[dao] trait JdbcLedgerDaoTransactionsSpec extends OptionValues with Insid
 
       // `pageSize = 2` and the offset gaps in the `commandWithOffsetGaps` above are to make sure
       // that streaming works with event pages separated by offsets that don't have events in the store
-      ledgerDao <- createLedgerDao(
+      response <- createLedgerDaoResourceOwner(
```

Comment:
original usage leaked a LedgerDao, with an active HikariCP inside, preventing user removal at test teardown (Oracle does not allow to remove a user if there are open connections to it apparently)